src/qofonoobject.cpp

```diff
@@ -240,6 +240,9 @@ void QOfonoObject::updateProperty(const QString &key, const QVariant &value)
             d_ptr->properties.remove(key);
         }
         propertyChanged(key, newValue);
+    } else if (!oldValue.isValid() && newValue.isValid()) {
```

Comment:
Why not? The property was missing, then it becomes valid, it's a property change, isn't it?